MANIFEST.in

```diff
@@ -1,9 +1,42 @@
 # Add documentation to the source tarball but not in the installation
-include doc/Makefile doc/.nojekyll doc/conf.py LICENSE.txt
+
+include doc/Makefile doc/.nojekyll doc/conf.py doc/CNAME
 include doc/*.rst
-include README.rst
-include vispy.json
 recursive-include doc/ext *
 recursive-include doc/_static *
 recursive-include doc/_templates *
+
 recursive-include vispy/static *.*
+recursive-include vispy *.py
+recursive-include vispy *.pyx
+recursive-include vispy *.glsl
+recursive-include vispy *.frag
+recursive-include vispy *.vert
+
+recursive-include js *.*
```

Comment:
No! No! No! No! This ends up adding all `node_modules`.

Reply:
Wasn't sure the right thing to do here

Reply:
Yeah, I'm trying to figure out what might be best here. Was just surprised when building the sdist when a ton of JS libraries were added. 😉

Reply:
FWIW the wheels are only 1.5 MB IIRC so it's not so bad